en/how-to-publish/iati-organisation-identifiers.rst

```diff
@@ -0,0 +1,41 @@
+IATI Organisation Identifiers
+-----------------------------
+In IATI, publishers are identified by their IATI org ID. You can form your IATI org ID by following the steps below. These steps should ensure that your ID unique and will be approved by the IATI tech team.
+
+**1) Identify your registration agency**
+
+A list of registration agencies can be found at: http://org-id.guide/ Check whether your organisation is registered with one of the agencies on this list. If yes, please use the code provided as the first part of your IATI organisation identifier.
+
+If you cannot find the agency your organisation is registered with then please send the name and website of the agency to: contact@org-id.guide. The org-ID team will then start the process of setting up a code for your registration agency.
+
+Example:
+
+  Companies House is a registration agency in the UK. It’s code is: 
```

Comment:
I wonder if this could be make clearer... Maybe if we say...

> Example 1: Company A is registered in the UK
> Companies House is a registration agency for companies in the UK. It’s code is `GB-COH`. Therefore the organisation identifier for company A will begin with `GB-COH-`

> Example 2: Charity B is registered in the Kenya
> [Similar example as above]

> Example 3: Government Agency C is registered in the USA
> [Similar example as above]